src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objectOfStyle = {};
+
+  if (sourceString.trim() === '') {
+    return {};
+  }
+
+  const arrFromString = sourceString.trim().split(';');
+
+  const getKeyandProp = arrFromString.map((item) => {
+    const [key, prop] = item.split(':').map((part) => part.trim());
+
+    return { key, prop };
+  });
+
+  getKeyandProp.forEach(({ key, prop }) => {
+    if (key !== '') {
+      objectOfStyle[key] = prop;
+    }
+  });
```

Comment:
The 'forEach' loop here is unnecessary since you can perform the assignment inside the 'map' function. This would also adhere to the checklist point of avoiding loops and using iteration methods instead.